### PR TITLE
#154953729 presenting deactivated users in gym overview

### PR DESCRIPTION
**Pull request by Jak-Sidious**
#### What does this PR do?

This PR enables for both active and inactive users to be displayed in the gym overview.

#### Description of Task to be completed?

- The tasks completed involve the creation of new dictionary keys to cater for the storage of active and inactive users. 
- Creation of new templates to render and display the separate tables for both active and inactive users
- Adding the tab functionality to the gym overview play to display all tables at first and offer the ability to hide or show respective gym users 

#### How should this be manually tested?

This can be manually tested by checking the gym overview page and viewing the various gym status of each user.

#### Any background context you want to provide?

By default, Gym users are either active or inactive but originally, you needed to view the gym member to view their status

#### What are the relevant pivotal tracker stories?

#154953729

#### Screenshots (if appropriate)

1. Selecting a gym out of the many available

gym selection

2. Viewing all Gym members

all

3. Viewing active members

active

4. Viewing inactive members

inactive

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Sebuliba-Adrian) [inline .vscode/settings.json]: I don't think we should have a .vscode in here

File: wger/gym/templates/gym/active_user_list.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<script>
+$(document).ready( function () {
+    /* Make table sortable */
+    $('#active_member_list').DataTable({
+        paging: false,
+        bFilter: true,
+        bInfo : false
+    });
+});
+</script>
```
Comment (Sebuliba-Adrian) [inline wger/gym/templates/gym/active_user_list.html]: You can always put this in the js file in static

File: wger/gym/templates/gym/inactive_user_list.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<script src="{% static 'bower_components/datatables/media/js/jquery.dataTables.min.js' %}" ></script>
+<script src="{% static 'bower_components/datatables/media/js/dataTables.bootstrap.min.js' %}" ></script>
+<script>
+$(document).ready( function () {
+    /* Make table sortable */
+    $('#inactive_member_list').DataTable({
+        paging: false,
+        bFilter: true,
+        bInfo : false
+    });
```
Comment (Sebuliba-Adrian) [inline wger/gym/templates/gym/inactive_user_list.html]: same here

File: wger/gym/templates/gym/member_list.html [hunk 4]
```diff
@@ -323,4 +347,23 @@ <h4>{% trans "Emails" %}</h4>
...
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+    }
+    document.getElementById(cityName).style.display = "block";
+    evt.currentTarget.className += " active";
+}
+
+// Get the element with id="defaultOpen" and click on it
+document.getElementById("defaultOpen").click();
+</script>
+
```
Comment (Sebuliba-Adrian) [inline wger/gym/templates/gym/member_list.html]: here too

File: .vscode/settings.json (hunk 1 above)
Comment (Jak-Sidious) [reply .vscode/settings.json]: This change has been made accordingly

File: wger/gym/templates/gym/member_list.html (hunk 4 above)
Comment (Jak-Sidious) [reply wger/gym/templates/gym/member_list.html]: I was unable to move the Jquery for this particular functionality into a different file

**Comment by Jak-Sidious** [comment]
As far as some of the requested changes go, specifically those to do with the moving of the javascript files they cannot be moved as moving them creates linting errors

**Comment by Sebuliba-Adrian** [comment]
@Jak-Sidious I have seen the changes that you have made. Every thing looks good to me. And for the linting errors that come as a result moving some files, you may live them as is for now.

**Pull request merged**